packages/jest-preset-kyt-enzyme/jest-preset.js

```diff
@@ -1,6 +1,6 @@
 const jestConfig = {
   verbose: true,
-  moduleFileExtensions: ['js', 'jsx', 'json'],
+  moduleFileExtensions: ['js', 'jsx', 'json', 'ts', 'tsx'],
```

Comment:
Here's [Jest's default `moduleFileExtensions` config](https://jestjs.io/docs/en/configuration.html#modulefileextensions-arraystring) for reference:

```json
{
  "moduleFileExtensions": ["js", "json", "jsx", "ts", "tsx", "node"]
}
```